apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDbType.cs

```diff
@@ -177,6 +177,14 @@ public DbType DbType
             }
         }
 
+        /// <summary>
+        /// Gets the configured <see cref="System.Type"/> for this SpannerDbType based on <see cref="SpannerConversionOptions"/>.
+        /// If the configuration for this type doesn't exist in options, the <see cref="DefaultClrType"/> is returned.
+        /// </summary>
+        /// <param name="options">Options to configure the CLR Type for this SpannerDbType.</param>
+        /// <returns>The configured <see cref="System.Type"/>.</returns>
+        internal System.Type GetConfiguredClrType(SpannerConversionOptions options) => DefaultClrType;
```

Comment:
I'd move the implementation to this method now, and have `DefaultClrType` call this - that way when we implement an option, there's less code to change.

We might want to clarify the docs for DefaultClrType, too.

Reply:
Sure. Will do so.

Reply:
Have moved the implementation to method. Could you please help with what I need to update in docs for `DefaultClrType`?